### PR TITLE
Testnet links for uptime

### PR DESCRIPTION
**Pull request by StanislavBreadless**

### CODE REVIEW

**Review by astarinmymind — COMMENTED** [review]
typo: mainnet

**Review by popzxc — APPROVED** [review]

File: docs/uptime.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Uptime
+
+## Mainnnet
```
Comment (astarinmymind) [inline docs/uptime.md]: typo mainnet

**Review by astarinmymind — APPROVED** [review]
lgtm

**Pull request merged**